### DISCUSSION TITLE
How to overide redirect_to ?

### DISCUSSION THREAD

**Discussion by Tioneb12** [janko/rodauth-rails#334] [Q&A]
On a public page (/post/:id), I have a login button (rodauth.login_path).

I would like to redirect to this page after logging in, but I'm always redirected to the homepage.

How can I achieve this?

**Comment by janko** [comment]
See the second example in [this guide](https://rodauth.jeremyevans.net/rdoc/files/doc/guides/login_return_rdoc.html).
